index.html

```diff
@@ -417,42 +384,57 @@
     };
 
     callback ThingEventListener = void (ConsumedThing thing);
+
+    enum ConsumedThingEvents {
+        "propertychanged",
+        "propertyremoved",
+        "propertyadded",
+        "actionchanged"
```

Comment:
yes, sorry, typoed.
the parametrized event makes sense for me, but I'd think it could even lead to "interfacechanged" (or maybe "TDchanged") with the parameter like `{"method" : "add", "type" : "action", "name" : "bla"}`

Not sure which one is better: to have many simple events or fewer parameterized ones.